word_guess.rb

```diff
@@ -0,0 +1,163 @@
+#word guess game
+
+#
+require "colorize"
+
+class Word_guess
+	attr_accessor :wrong_count, :word_list, :secret_word, :game_word, :game_word_array
+
+	def initialize
```

Comment:
I'm really happy with how you've structured these methods. It's great. Similarly, your code formatting (spacing, indentation) is pretty spot on. Moving forward, I'm going to ask the class to standardize on two-space indents. Looks like you're using eight-space indents here which feels excessive.